### ISSUE TITLE
Support for namespaces?

### ISSUE THREAD

**Issue by Gekke-Jakke** [larshp/abapmerge#28]
When I try to build a simple project using a namespace, I get following issue:
![image](https://user-images.githubusercontent.com/2744548/29427915-9d70da24-838b-11e7-9378-43ca7ebe0b2b.png)

But the include is defenitely used. Is there any way to get this working with namespaces?

**Comment by larshp** [comment]
it is just code, so it is possible to get it working, sure

just wondering, what is your usecase for abapmerge? If using abapGit then abapmerge should not be needed? Perhaps you want to distribute your code without using abapGit?

**Comment by larshp** [comment]
can you help making a public dummy project where it fails with a similar error?

**Comment by Gekke-Jakke** [comment]
We are building solutions on our rollout template system (that's why we're using namespaces), the customer won't be able to modify the programs (we don't want that, because we'll be providing updates for our solutions). 

By having a "build" version of some of our solutions we are able to provide the customer the source code that they can use in a Z-report very easily (and they can do whatever they like with the reports). Another plus, we can even reuse the code to provide it to customers who don't want our namespace and / or abapgit installed in their system.

I've made the dummy project that I used for testing the abapMerge project public and added you as a member. (It's on Gitlab )
https://gitlab.com/Gekke-Jakke/dummy-project/
